### PR TITLE
add note about enabling Sandbox using Powershell

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9098

### CODE REVIEW

File: windows/security/threat-protection/windows-sandbox/windows-sandbox-overview.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ The following video provides an overview of Windows Sandbox.
 1. Use the search bar on the task bar and type **Turn Windows Features on and off** to access the Windows Optional Features tool. Select **Windows Sandbox** and then **OK**. Restart the computer if you're prompted.
 
    - If the **Windows Sandbox** option is unavailable, your computer doesn't meet the requirements to run Windows Sandbox. If you think this is incorrect, review the prerequisite list as well as steps 1 and 2.
+
+>[!NOTE]
+> To enable Sandbox using Powershell, open Powershell as Administrator and run **Enable-WindowsOptionalFeature -FeatureName "Containers-DisposableClientVM" -All -Online**
```
Comment (illfated) [inline windows/security/threat-protection/windows-sandbox/windows-sandbox-overview.md:58]: For consistency, please add a MarkDown indent marker compatibility spacing here too:
```suggestion
> [!NOTE]
> To enable Sandbox using PowerShell, open PowerShell as Administrator and run **Enable-WindowsOptionalFeature -FeatureName "Containers-DisposableClientVM" -All -Online**
```
Also: The name "PowerShell" is usually written in CamelCase.

BTW, would it be interesting to encapsulate the command line in MarkDown back ticks, both to display it in monospaced typeface (font) and also shield it from being translated by Machine Translation used for this page when it is auto-translated to other languages?

Comment (JohanFreelancer9) [reply windows/security/threat-protection/windows-sandbox/windows-sandbox-overview.md:58]: Agree with these suggestions. Also please insert a period (.) at the end of the note.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/security/threat-protection/windows-sandbox/windows-sandbox-overview.md (hunk 1 above)
Comment (illfated) [reply windows/security/threat-protection/windows-sandbox/windows-sandbox-overview.md:58]: Done. Thank you for pointing that out. 👍🏻

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me. The only thing I would have considered adding to the existing PR content (knowing that I tend to be an OCD perfectionist) is a separate code block for the PS code, but that should not be necessary when the goal is just to inform the reader what the simple command content looks like (and it would seem like a bit of "overkill" to do so).

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**